test/base/TextTest.ooc

```diff
@@ -166,6 +166,13 @@ TextTest: class extends Fixture {
 			text free()
 			text2 free()
 		})
+		this add("trim", func {
+			paddedText := t"  \t test \n test \r\n\t "
+			trimmedText := paddedText trim()
+			expect(trimmedText == t"test \n test")
+			paddedText free()
```

Comment:
It is already freed by the comparison and on top of this these are literals so no need to free at all.